internal/server/server.go

```diff
@@ -94,6 +103,6 @@ func Run() error {
 		return fmt.Errorf("error while run server: %w", err)
 	}
 	<-waitForShutdown
-	fmt.Println("Server shutdowned gracefull")
+	fmt.Println("Server shutdowned graceful")
```

Comment:
```suggestion
	fmt.Println("Server was shut down gracefully")
```
это если уж мы стремимся к нормальному английскому...